mmdet3d/core/visualizer/show_result.py

```diff
@@ -4,8 +4,8 @@
 from os import path as osp
 
 
-def _write_ply(points, out_filename):
-    """Write points into ``ply`` format for meshlab visualization.
+def _write_obj(points, out_filename):
```

Comment:
Is renaming this function necessary?
If it is for correctness, may check the whole codebase and fix it completely.

Reply:
Seems the original point saving function uses wrong name, but do we need to use consistent format to save points and bboxes? Currently it seems we save points into `obj` format while save bboxes into `ply` format.

Reply:
I agree. Trimesh can easily export mesh into ```obj``` format, so it is easy to make them consistent here.